.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+    "editor.renderIndentGuides": true,
+    "php.validate.run": "onType",
+    "php.validate.executablePath": "/usr/bin/php"
+}
```

Comment:
Please gitignore the .vscode folder (and not commit this file as well) since everyone is using different IDEs and this is just clutter for most everyone else.